currency_rate_inverted/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 
 {
     "name": "Currency Rate Inverted",
-    "version": "13.0.1.0.1",
+    "version": "14.0.1.0.0",
```

Comment:
This version change should be done as a separate commit. See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

Reply:
I already corrected